docs/src/create_reference_md.py

```diff
@@ -170,6 +170,12 @@ def source_section(source: Source, source_key: str, level: int) -> str:
     if source.supported_versions_description:
         title = f"Supported {source.name} versions"
         markdown += admonition(source.supported_versions_description, title, "important")
+    if source.deprecated:
+        deprecation_message = (
+            f"Support for using {source.name} as source is deprecated. "
```

Comment:
Likewise, perhaps here we should add that it will be removed (/ is marked for removal) in a future version?

Reply:
I'd rather keep this text brief and generic and have the details in a GitHub issue. That's easier to change than the code.